src/registers/model_specific.rs

```diff
@@ -445,4 +479,41 @@ mod x86_64 {
             unsafe { msr.write(value.bits()) };
         }
     }
+
+    impl UCet {
+        /// Read the raw IA32_U_CET.
+        #[inline]
+        fn read_raw() -> u64 {
+            unsafe { Self::MSR.read() }
+        }
+
+        /// Write the raw IA32_U_CET.
+        #[inline]
+        fn write_raw(value: u64) {
+            let mut msr = Self::MSR;
+            unsafe {
+                msr.write(value);
+            }
+        }
+
+        /// Read IA32_U_CET.
+        #[inline]
+        pub fn read() -> (CetFlags, Page<Size4KiB>) {
```

Comment:
```suggestion
        pub fn read() -> (CetFlags, Page) {
```
`Page` defaults to 4k pages.